hudi-client/hudi-spark-client/pom.xml

```diff
@@ -43,6 +43,11 @@
       <artifactId>hudi-client-common</artifactId>
       <version>${project.parent.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hudi</groupId>
+      <artifactId>hudi-metastore</artifactId>
```

Comment:
can we call this `hudi-metaserver` instead? Also should we break the client and server of the metaserver into  different modules? Might be good to do it from first version. The spark-client should only need to depend on the `hudi-metaserver-client`

Reply:
metaserver and metastore you mean? we can split server and client components for now? as a starting point. We can have a multi-module setup like how `hudi-sync` is